neo/sys/posix/posix_main.cpp

```diff
@@ -688,11 +688,7 @@ Posix_Cwd
 const char* Posix_Cwd()
 {
 	static char cwd[MAX_OSPATH];
```

Comment:
It also breaks the current behaviour of `Posix_Cwd()` to return a pointer to a region of memory, that *always* contains the current working directory at the last time it was queried. I don't know, what the intended behaviour is, but it might break code calling `Posix_Cwd()` to deallocate the returned buffer at a later time.